### ISSUE TITLE
Give more info to redis/pymysql/pymongo

### ISSUE THREAD

**Issue by huangzhw** [open-telemetry/opentelemetry-python-contrib#2337]
**Is your feature request related to a problem?**
The old OpenTracing provided more information to us to solve question.
Redis only provides `SET ? ?`, it's not useful.
pymysql lacks parameters.
pymongo lacks command info and event.reply.

**Describe the solution you'd like**
Maybe we can provide some options like verbose to control the output.

**Comment by Kludex** [comment]
I agree. PyMongo at least has `capture_statement`. I think Redis should receive the same parameter.

**Comment by veritaspei** [comment]
i agree, is there any solution?
